scripts/jenkins/jenkinsfiles/Jenkinsfile-PrismaScan

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/groovy
+
+@Library('test-shared-library') _
+
+def dockerImage
+def branchOrTag
+def steamImage
+
+pipeline {
+    agent { node { label 'linux&&docker' } }
+
+    parameters {
+        string(name: 'BRANCH_OR_TAG', defaultValue: 'master', description: 'Enter branch or tag you want to scan.')
+    }
+
+    options {
+        ansiColor('xterm')
+        timestamps()
+    }
+
+    stages {
+        stage('0. Init'){
+            steps{
+                script{
+                    checkout(
+                        scm: [$class: 'GitSCM',
+                            branches: [[name: '${BRANCH_OR_TAG}'], [name: 'devops/steam-prisma-jenkins']],
+                            userRemoteConfigs: [[url: 'https://github.com/h2oai/h2o-3.git']]
+                        ]
+                    )
+
+                    sh "git checkout devops/steam-prisma-jenkins"
+                    
+                    dir("docker/prisma"){
+                        dockerImage = docker.build("node-java","-f Dockerfile .")
+                    }
+                }
+                
+            }
+        }
+        stage('1. Build jar') {
+            steps {
+                script{
+                    dockerImage.inside(){
+                        sh "git checkout ${BRANCH_OR_TAG}"
+                        sh "./gradlew :h2o-assemblies:steam:shadowJar"
+                        archiveArtifacts artifacts: "h2o-assemblies/steam/build/libs/*.jar"
+                    }
+                }
+            }
+        }
+        stage('2. Scan jar using Prisma'){
+            steps {
+                script{
+                    branchOrTag = "${BRANCH_OR_TAG}".replace('/','-')
+                    steamImage = "h2o-assemblies/steam:${BUILD_NUMBER}-${branchOrTag}"
+
+                    sh """
+                        git checkout devops/steam-prisma-jenkins
```

Comment:
Do we need this line? Is the `devops/steam-prisma-jenkins` going live forever?

Reply:
same here